setup/steps/configure_monitoring_user_password.go

```diff
@@ -0,0 +1,69 @@
+package steps
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"errors"
+	"fmt"
+
+	survey "github.com/AlecAivazis/survey/v2"
+	s "github.com/pganalyze/collector/setup/state"
+)
+
+var ConfigureMonitoringUserPasswd = &s.Step{
+	Description: "Configure monitoring user password",
+	Check: func(state *s.SetupState) (bool, error) {
+		hasPassword := state.CurrentSection.HasKey("db_password")
+		return hasPassword, nil
+	},
+	Run: func(state *s.SetupState) error {
+		var passwordStrategy int
+		if state.Inputs.Scripted {
+			if state.Inputs.GenerateMonitoringPassword.Valid && state.Inputs.GenerateMonitoringPassword.Bool {
+				if state.Inputs.Settings.DBPassword.Valid && state.Inputs.Settings.DBPassword.String != "" {
+					return errors.New("cannot specify both generate password and set explicit password")
+				}
+				passwordStrategy = 0
+			} else if state.Inputs.Settings.DBPassword.Valid && state.Inputs.Settings.DBPassword.String != "" {
+				passwordStrategy = 1
+			} else {
+				return errors.New("no db_password specified and generate_monitoring_password flag not set")
+			}
+		} else {
+			err := survey.AskOne(&survey.Select{
+				Message: "Select how to set up the collector user password (will be saved to collector config):",
+				Options: []string{"generate random password (recommended)", "enter existing password"},
```

Comment:
Maybe `enter password` (instead of `enter existing password`, since we can set the password for you)